installing_on_kubernetes/_topics/checking_readiness.md

```diff
@@ -0,0 +1,41 @@
+## Check readiness of the MIQ pods
+
+_**Note:**_ Please allow a few minutes for the application to start.  Initial install and upgrades take extra time due do database initialization.  HTTPS traffic will not be served until pods are running and ready.  The READY column denotes the number of replicas and their readiness state.  The operator deploys the following pods:
+
+```bash
+$ oc get pods
```

Comment:
Not for this PR, but we should switch to `kubectl` to avoid being openshift specific.